index.js

```diff
@@ -19,7 +19,11 @@ module.exports = function (params) {
             childArgs = [];
 
         if (options['phantomjs-options'] && options['phantomjs-options'].length) {
-            childArgs.push(options['phantomjs-options']);
+            if (typeof options['phantomjs-options'] === 'string') {
```

Comment:
Instead of checking for a string, could we check to see whether it is an array or not? Example,
```js
if (!Array.isArray(options['phantomjs-options'])) {}
```